lightkurve/search.py

```diff
@@ -0,0 +1,531 @@
+from __future__ import division, print_function
+import os
+import logging
+import numpy as np
+
+from astroquery.mast import Observations
+from astroquery.exceptions import ResolverError
+from astropy.table import unique, join
+from astropy.coordinates import SkyCoord
+from astropy.io import ascii
+from .lightcurvefile import KeplerLightCurveFile
+from .targetpixelfile import KeplerTargetPixelFile
+from .collections import TargetPixelFileCollection, LightCurveFileCollection
+
+from . import PACKAGEDIR
+log = logging.getLogger(__name__)
+
+
```

Comment:
Let's enable `from lightkurve import search_targetpixelfile` (rather than `from lightkurve.search import ...`) by doing this:

```suggestion
__all__ = ['search_targetpixelfile', 'search_lightcurvefile']

```